products/public-websites/README.md

```diff
@@ -582,7 +583,7 @@ VA.gov uses search.gov as its engine. Public Websites owns that integration, and
 * [On Site Search - product docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/on-site-search)
 * Design system: [Search input](https://design.va.gov/components/search-input)
 * Search.gov indexing documentation: https://search.gov/indexing/indexing-with-searchgov.html 
-* 2020: bring more sites under search banner: [https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/search](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/search) 
+* 2020: bring more sites under search banner: [https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/search](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/search) TODO dead link
```

Comment:
The `global` folder no longer exists. I poked around for a while trying to find docs that could fit here. What is the new link for this?

Reply:
Let's remove this line. It's a dupe of 3 bullets up.

Reply:
👍